docs/experimentation.md

```diff
@@ -1,27 +1,71 @@
 # Experimentation
 
 ## Introduction
-In a nutshell, **Experimentation** module enables to run A/B/../n testing for your configurations in equi-sized groups/cohorts, it works on top of **Context-Aware-Config** and can also work as a release system for your configuration.
+
+The **Experimentation** component enables one to run A/B/../Z testing for your
+configurations in equi-sized groups/cohorts. It works on top of
+**Context-Aware-Config**.  The Experimentation component can also be used as a
+release system for your configuration changes which becomes an A/B test with
+two variants - the current value and new value.
 
 ## Concepts
 
 ### Experiment
-An experiment as name suggest enables you to test and evaluate the behaviour of the system for different values of the same configuration. An experiment can have exactly one **CONTROL** variant and `n` **EXPERIMENTAL** variants, with each variant overriding/changing same set of keys in configuration. An experiment's scope can be controlled by declaring the context, which chalks out the sample set for the experiment from the population.
-
-<br/>
-<br/>
+An experiment, as the name suggests, enables you to test and evaluate the
+behaviour of the system for different values of the same configuration. An
+experiment can have exactly one **CONTROL** variant and `n` **EXPERIMENTAL**
+variants, with each variant overriding/changing same set of keys in
+configuration. An experiment's scope can be controlled by declaring the
+context, which chalks out the sample set for the experiment from the
+population.
 
-![context-example](experiment-context-example.png)
```

Comment:
@ShubhranshuSanjeev  - pardon me for removing the diagrams - wanted to refer to the same example as the proposed demo app - hence took the liberty.